tests/system/src/test/java/test/util/Util.java

```diff
@@ -293,4 +299,100 @@ public static ArrayList<String> createApplicationLaunchCommand(
 
         return cmd;
     }
+
+    /**
+     * Launches an FX application, at the same time ensuring that it has been
+     * actually launched within {@link #STARTUP_TIMEOUT} (15 seconds).
+     * <p>
+     * The application being started must call {@link CountdownLatch#countDown()} once to signal
+     * its successful start (for example, by setting a handler for {@link javafx.stage.WindowEvent.WINDOW_SHOWN} event
+     * on its primary Stage).
+     *
+     * @param startupLatch - a latch used to communicate successful start of the application
+     * @param applicationClass - application to launch
+     * @param args - command line arguments
```

Comment:
Since we don't generate javadocs for our tests, this doesn't matter, but for future refence, there should not be any punctuation (i.e., no `-`) after the name of the parameter.

Reply:
Got it, thanks.